src/lib/connectAggregationToTransform.js

```diff
@@ -27,6 +27,10 @@ export default function connectAggregationToTransform(WrappedComponent) {
 
     getChildContext() {
       return {
+        getValObject: attr =>
```

Comment:
here and in the next few files I'm just making sure each `connectXToY` properly rewraps the top-level `getValObject` calls.